javascript/modules/client/README.md

```diff
@@ -180,16 +179,37 @@ for await (const step of client.methods.deposit(depositParams)) {
 
 ### Loading DAO details
 
-- __TODO__:  Name, metadata, installed plugin list
+Handles retrieving DAO metadata using unique identifier.
+
+```ts
+import { Client } from "@aragon/sdk-client";
+
+const client = new Client(context);
+const daoIdentifier = "0x1234..."; // unique identifier; dao name or address
+
+const metadata = await client.methods.getDaoMetadata(daoIdentifier);
+console.log(metadata);
+
+/* 
+{
+   address: "0x1234...",
+   avatar: "http..."
+   createdAt: 1656411653742,
+   description: "This dao...",
+   links: [{label: "Website", url: "http..."}];
+   name: "Abc Dao",
+   packages: ["ERC20VotingPackage"];
```

Comment:
That will probably change in the future, but for now, I would return the registry ID of the plugins installed
```suggestion
   packages: ["0x1234..."],
```